cve_bin_tool/checkers/glibc.py

```diff
@@ -17,14 +19,59 @@ class GlibcChecker(Checker):
     CONTAINS_PATTERNS = [
         r"The following command substitution is needed to make ldd work in SELinux",
         r"environments where the RTLD might not have permission to write to the",
+        r"Valid options for the LD_DEBUG environment variable are:",
+        r"To direct the debugging output into a file instead of standard output",
+        r"Mandatory or optional arguments to long options are also mandatory or optional for any corresponding short options.",
     ]
     FILENAME_PATTERNS = [
         r"ldd",
         r"libc6",
         r"glibc-source",
         r"glibc",
         r"nscd",
-        r"libc.so.",
+        r"libc\.so\.6",
```

Comment:
```suggestion
        r"libc\.so\.",
```

I don't know that we correctly detect versions that predate the 6, but we want to be able to give people a hint if one is found.